executor/common_fuchsia.h

```diff
@@ -176,13 +176,22 @@ long syz_mmap(size_t addr, size_t size)
 		fail("zx_object_get_info(ZX_INFO_VMAR) failed: %d", status);
 	zx_handle_t vmo;
 	status = zx_vmo_create(size, 0, &vmo);
-	if (status != ZX_OK)
+	if (status != ZX_OK) {
+		fprintf(stderr, "zx_vmo_create failed with: %d", status);
```

Comment:
This syscall is also available to fuzzer:
https://github.com/google/syzkaller/blob/master/sys/fuchsia/sys.txt#L16
and it will sure pass args that will make it fail. Printing output during fuzzing is bad, it will slow down process, can overflow pipes and in the end nobody will see it (when you run syz-manager you don't see syz-fuzzer output, and even if you see syz-fuzzer output, it does not include syz-executor output).

I think we should replace printf with debug. If the initial syz_mmap coming from os_init will fail, it should loudly fail everything and then one can run syz-execprog with -debug flag and see this additional output (if exact root cause is not obvious through other means).